marklogic-data-hub/src/testFixtures/java/com/marklogic/hub/test/Action.java

```diff
@@ -0,0 +1,11 @@
+package com.marklogic.hub.test;
+
+/**
+ * Represents an operation that accepts no input arguments and returns no
+ * result.
+ */
+@FunctionalInterface
+public interface Action {
+    
+    void execute();
```

Comment:
Note that when you do need a no-arg no-return-value interface in the future, you can just use Runnable instead.

Reply:
I did not use Runnable as it is associated with threads and might cause confusion in the future.